### PR TITLE
Contact Support: Update RELEASE-NOTES.txt

### LINKED ISSUE #21365: Update RELEASE-NOTES iOS

### PR DESCRIPTION
**Pull request by staskus**
Fixes #21365

Preparing for 23.2 release

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@
 * [*] [internal] Change how a post is fetched when selecting a post in Spotlight search result. [#21439]
 * [*] [internal] Change how a post's revision is fetched from the post history screen. [#21440]
 * [**] [internal] Replace the progress indicator implementation in uploading featured image from "Post Settings" [#21438]
+* [***] [Jetpack-only] Contact Support: Need assistance? Use Jetpack mobile assistant for immediate help. Create a support ticket for further assistance. [#21467]
```
Comment (guarani) [inline RELEASE-NOTES.txt]: I'm not sure if this is descriptive enough for someone unfamiliar with the feature to know what it is (e.g. the editorial team or folks testing the beta).

May I suggest an alternative? The same suggestion applies to Android @ravishanker.

> Contact Support: Add a new chat-based support channel where users can get answers from a bot trained to help app users. Users can still create a support ticket to talk to a Happiness Engineer if they don't find the answer they're looking for [#21467]

**Review by guarani — COMMENTED** [review]
Hi @staskus, I'd love to hear your thoughts on the below suggestion 🙇

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (guarani) [reply RELEASE-NOTES.txt]: Some additional context: pdcxQM-Km-p2

Comment (staskus) [reply RELEASE-NOTES.txt]: Thanks! I agree your suggestion sounds more descriptive of what the feature actually does. I would accept your suggestion 👍

Comment (irfano) [reply RELEASE-NOTES.txt]: I agree with the suggestion and updated the Android release notes: https://github.com/wordpress-mobile/WordPress-Android/pull/19124

**Review by irfano — APPROVED** [review]

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (guarani) [reply RELEASE-NOTES.txt]: Thanks folks!

**Pull request merged**